player/command.c

```diff
@@ -2548,7 +2548,10 @@ static int mp_property_video_frame_info(void *ctx, struct m_property *prop,
             .fps = lrint(mpctx->vo_chain->filter->container_fps),
         };
         int frame = lrint(get_current_pos_ratio(mpctx, false) * get_frame_count(mpctx));
-        av_timecode_make_string(&tcr, approx_smpte, frame);
+        // Avoid division-by-zero in av_timecode_make_string() if reported
+        // container_fps is or rounds to 0.
+        if (tcr.fps)
```

Comment:
We could check this earlier to avoid all other `get_...` 

Would `if (s12m_tc[0] == '\0' && mpctx->vo_chain && mpctx->vo_chain->filter->container_fps > 0) {` work for you?

Reply:
Of course. I wrote something similar first (incoming), but went for a cleaner diff later.